src/promptflow/promptflow/_core/_errors.py

```diff
@@ -1,7 +1,13 @@
 from traceback import TracebackException
 
 from promptflow._utils.exception_utils import ADDITIONAL_INFO_USER_EXECUTION_ERROR, last_frame_info
-from promptflow.exceptions import ErrorTarget, UserErrorException, ValidationException
+from promptflow.exceptions import ErrorTarget, SystemErrorException, UserErrorException, ValidationException
+
+
+class NotSupportedError(SystemErrorException):
```

Comment:
`NotSupported` would be a good error code.

Reply:
Updated.